gui/storage/models.py

```diff
@@ -753,16 +753,6 @@ def __init__(self, *args, **kwargs):
         super(Disk, self).__init__(*args, **kwargs)
         self._original_state = dict(self.__dict__)
```

Comment:
It seems original state is not used anywhere? Whole `__init__` could be removed.